product_variant_standard_price_update/README.rst

```diff
@@ -0,0 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-LGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/lgpl-3.0-standalone.html
+   :alt: License: LGPL-3
+
+=====================================
+Product variant standard price update
+=====================================
+
+This module extends the product variant feature to allow update standard price
+in all variants from the product template form
+
+Installation
+============
+
+The module adds a button in template form for show a wizard to update the
```

Comment:
But it's a lot of code to maintain that duplicates a function that already exists. And I don't agree it's more UX, as you have the same control and easiness with the other method.